### DISCUSSION TITLE
Focus on incremental Webpack builds instead of Vite?

### DISCUSSION THREAD

**Discussion by rfox12** [quasarframework/quasar#11464] [Ideas / Proposals]
I know @rstoenescu has been working on Vite (which uses rollup); but in my testing, incremental Webpack builds should, as far as I can tell, yield a faster change-re-run development cycle.  And Webpack is a fundamental part of Quasar for me.  There are things in Webpack like federated module integration that is really a core killer feature.  Has there been any attempts or discussions about getting incremental Webpack builds ("[watch](https://webpack.js.org/configuration/watch/)") working with Quasar?

Personally I gave it a shot -- quasar.conf to add "webpack watch" options in a Quasar _build_ (not dev) almost works but for a few lines in the quasar build script that needed to be commented out (because the build script is run on every change):

Specifically in https://github.com/quasarframework/quasar/blob/dev/app/bin/quasar-build
if I remove
```
    // free up memory
    webpackData = void 0
```
Then it mostly works.  With HMR in **dev**, and incremental builds in **build**, wouldn't Webpack still be faster / better than anything Vite can offer?

**Comment by hawkeye64** [comment]
@rfox12 If you are interested in webpack speed, also try this in your quasar.conf.js -> build

https://github.com/hawkeye64/quasar-extras-svg-icons/blob/main/docs/quasar.conf.js#L109-L113
